### ISSUE TITLE
Prediction results

### ISSUE THREAD

**Issue by SonwYang** [tist0bsc/SGCN#1]
I train this model according to the default configuration. When the training is done,  I run the prediction code and found that the the visualization of prediction is bad. Can you provide your best model params?

**Comment by tist0bsc** [comment]
Sorry, I upload the wrong dataset version, the validation set should be test set while the test set should be validation set. And I lost my previous model file, so I train a new one with 100 epochs. I put the best SGCN model (.pth), evalution results , visualization results, log of new val set on https://pan.baidu.com/s/1ZiQcqEfdJa5HruxsUYobGQ , code:1234  and the dataset on Google Drive has been updated.

**Comment by tist0bsc** [comment]
> I train this model according to the default configuration. When the training is done, I run the prediction code and found that the the visualization of prediction is bad. Can you provide your best model params?

Sorry, I upload the wrong dataset version, the validation set should be test set while the test set should be validation set. And I lost my previous model file, so I train a new one with 100 epochs. I put the best SGCN model (.pth), evalution results of new test set , visualization results of new test set, log file of new val set on https://pan.baidu.com/s/1ZiQcqEfdJa5HruxsUYobGQ , code:1234 and the dataset on Google Drive has been updated.

**Comment by lyf6** [comment]
@SonwYang have you repeated this result?

**Comment by BetterSgc** [comment]
老师你好，我运行了这里的train.py脚本，但是并没有发现saved文件夹，该如何操作？

**Comment by BetterSgc** [comment]
老师你好，我运行了这里的train.py脚本，但是并没有出现saved文件夹，我该如何操作？

**Comment by yjwong1999** [comment]
Hi @tist0bsc 

Can you share the pretrained model weights via google drive link? I'm unable to access your pretrained model via the baidu drive, because I couldn't register an account 🥲

**Comment by 666305** [comment]
@SonwYang @yjwong1999  have you repeated this result?
